docs/src/pages/components/snackbars/snackbars.md

```diff
@@ -108,4 +108,28 @@ It also enables you to **stack** them on top of one another (although this is di
 
 (WAI-ARIA: https://www.w3.org/TR/wai-aria-1.1/#alert)
 
-- By default, the snackbar won't auto-hide. However, if you decide to use the `autoHideDuration` prop, it's recommended to give the user [sufficient time](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits.html) to respond.
+By default, the snackbar won't auto-hide. However, if you decide to use the `autoHideDuration` prop, it's recommended to give the user [sufficient time](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits.html) to respond.
+
+When open, **every** `Snackbar` will be dismissed if <kbd>Escape</kbd> is pressed.
```

Comment:
The `kbd` doesn't have the correct style. Could we add `class="key"`?

<img width="861" alt="Screenshot 2021-11-25 at 20 56 38" src="https://user-images.githubusercontent.com/3165635/143495066-5696a48e-c535-4c09-9d41-285830260e03.png">